api/proto/banyandb/database/v1/schema.proto

```diff
@@ -50,7 +50,7 @@ message TagSpec {
   // indexed_only indicates whether the tag is stored
   // True: It's indexed only, but not stored
   // False: it's stored and indexed
-  bool indexed_only = 3;
+  bool indexed_only = 3 [deprecated = true];
```

Comment:
@hanahmily As we don't keep forward compatibility, once Kai tested, should we remove this permanently, and use id=3 later for other things?